lib/nrf_modem_lib/nrf91_sockets.c

```diff
@@ -1133,17 +1138,9 @@ NET_DEVICE_OFFLOAD_INIT(nrf91_socket, "nrf91_socket",
 			0, &nrf91_iface_offload_api, 1280);
 
 #if defined(CONFIG_NRF_MODEM_LIB_NETIF)
-/* Bind l2 connectity APIs. */
-static struct conn_mgr_conn_api conn_api = {
-	.init = nrf_modem_lib_netif_init,
-	.connect = nrf_modem_lib_netif_connect,
-	.disconnect = nrf_modem_lib_netif_disconnect,
-	.set_opt = nrf_modem_lib_netif_options_set,
-	.get_opt = nrf_modem_lib_netif_options_get,
-};
-
+extern struct conn_mgr_conn_api conn_api;
 CONN_MGR_CONN_DEFINE(NRF_MODEM_LIB_NETIF, &conn_api);
 CONN_MGR_BIND_CONN(nrf91_socket, NRF_MODEM_LIB_NETIF);
-#endif /* CONFIG_NRF_MODEM_LIB_NETIF */
```

Comment:
nit: I personally find these closure comments `/* CONFIG_NRF_MODEM_LIB_NETIF */` helpful, even for short `#if` spans

Reply:
ok, I can add it back.